sandbox/auth/internal/authflow/authflow.go

```diff
@@ -48,16 +49,21 @@ func (f *AuthFlow) OpenBrowser() error {
 	return browser.OpenURL(f.URL)
 }
 
-func (f *AuthFlow) Exchange(code string) (*oauth2.Token, error) {
+func (f *AuthFlow) Exchange(code string) (*session.Token, error) {
 	ctx := context.Background()
-	tok, err := f.oauth2Conf.Exchange(ctx, code, pkce.VerifierOption(f.pkceCodeVerifier))
+	otok, err := f.oauth2Conf.Exchange(ctx, code, pkce.VerifierOption(f.pkceCodeVerifier))
 	if err != nil {
 		return nil, err
 	}
 
-	err = f.verify(tok)
+	tok, err := session.FromOauth2(otok)
 	if err != nil {
 		return nil, err
 	}
+
+	// err = f.verify(tok)
```

Comment:
no verify?

Reply:
I'm in the middle of re-implementing the token verification. I will re-enable it in a future change.